apps/artemis-web/components/hijack-table/hijack-table.tsx

```diff
@@ -183,23 +218,38 @@ const HijackTableComponent = (props) => {
 
   if (HIJACK_DATA && HIJACK_DATA.length) {
     hijacks = HIJACK_DATA.map((row) => ({
-      update: row.time_last,
-      time: row.time_detected,
+      update: formatDate(new Date(row.time_last)),
+      time: formatDate(new Date(row.time_detected)),
       hprefix: row.prefix,
       mprefix: row.configured_prefix,
-      type: row.type,
+      htype: row.type,
+      status: row.status,
       as: row.hijack_as,
-      rpki: row.key,
+      rpki: row.rpki_status,
       peers: row.num_peers_seen,
       ASes: row.num_asns_inf,
-      ack: row.seen,
-      more: row.comment,
+      ack:
+        row.resolved || row.under_mitigation ? (
+          <img src="./handled.png" />
+        ) : (
+          <img src="./unhadled.png" />
+        ),
+      more: <a href={'/hijack?key=' + row.key}>View</a>,
```

Comment:
If we want to visit this page with SSR, would it make sense to just pass the components as props?

Saying that I want both ones to be supported, so you either pass it from the parameter in the URL or from the component prop.